maro/cli/grass/executors/grass_executor.py

```diff
@@ -218,29 +222,68 @@ def remote_update_image_files_details(self):
 
     def remote_update_node_status(self, node_name: str, action: str):
         command = (
-            "ssh -o StrictHostKeyChecking=no "
+            f"ssh -o StrictHostKeyChecking=no -p {self.ssh_port} "
             f"{self.admin_username}@{self.cluster_details['master']['public_ip_address']} "
             f"'cd {GlobalPaths.MARO_GRASS_LIB}; python3 -m scripts.update_node_status "
             f"{self.cluster_name} {node_name} {action}'"
         )
         _ = SubProcess.run(command)
 
-    def test_connection(self, node_ip_address: str):
+    def test_ssh_22_connection(self, node_ip_address: str):
```

Comment:
Why have this test separately?

Reply:
We use 22 as ssh port by default, but users can assign a new port to it.
And when we are using grass/azure mode, we will have to replace port 22 with the new port manually, so we have a test_ssh_22 here and a test_ssh_new_port below